src/Services/BeamService.php

```diff
@@ -210,6 +210,12 @@ public function claim(string $code, string $wallet): bool
     public static function claimsCountResolver(string $code): Closure
     {
         return function () use ($code) {
+            if (self::isSingleUse($code)) {
+                $singleUseClaim = BeamClaim::withSingleUseCode($code)->first();
+
+                return !isset($singleUseClaim) || $singleUseClaim->claimed_at ? 0 : 1;
```

Comment:
I think this whole expression can be simplified with this
```suggestion
                return (int) empty($singleUseClaim?-> claimed_at )```